antarest/study/model.py

```diff
@@ -43,26 +44,44 @@
 
 DEFAULT_WORKSPACE_NAME = "default"
 
-STUDY_REFERENCE_TEMPLATES: t.Mapping[str, str] = {
-    "600": "empty_study_613.zip",
-    "610": "empty_study_613.zip",
-    "640": "empty_study_613.zip",
-    "700": "empty_study_700.zip",
-    "710": "empty_study_710.zip",
-    "720": "empty_study_720.zip",
-    "800": "empty_study_803.zip",
-    "810": "empty_study_810.zip",
-    "820": "empty_study_820.zip",
-    "830": "empty_study_830.zip",
-    "840": "empty_study_840.zip",
-    "850": "empty_study_850.zip",
-    "860": "empty_study_860.zip",
-    "870": "empty_study_870.zip",
-    "880": "empty_study_880.zip",
+NEW_DEFAULT_STUDY_VERSION: StudyVersion = StudyVersion.parse(880)
+STUDY_VERSION_600 = StudyVersion.parse("600")
```

Comment:
I would propose to:
- respect the dots also in variables names for example with underscors
- remove the patch version which is not useful
- use a true version string in the right hand side

Otherwise we will not be able to keep the same syntax for longer versions (8.10 ...).
This will make clear that study versions are only major.minor versions

```python
STUDY_VERSION_6_1 = StudyVersion.parse("6.1")
```